notebooks/dask-quickstart.ipynb

```diff
@@ -0,0 +1,1464 @@
+{
```

Comment:
Hrm, it seems odd to me to have two versions of the same material.  Does this mean that we need to update each every time that we want to make a change?